### PR TITLE
feature: highlighted ecosystem pools section

### PR DESCRIPTION
**Pull request by 1saf**
# Description
As part of the ecosystem pools project we want to show that we have partner pools on our Home page

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Other

## How should this be tested?

Test both mobile and desktop that there are ecosystem pools on the home page

## Checklist:

- [X] I have performed a self-review of my own code
- [X] I have commented my code where relevant, particularly in hard-to-understand areas
- [X] My changes generate no new console warnings
- [X] The base of this PR is `master` if hotfix, `develop` if not

### CODE REVIEW

**Comment by evgenyboxer** [comment]
@1saf could you please fix lint issues

**Review by garethfuller — APPROVED** [review]

**Comment by TomAFrench** [comment]
Should we display Element.fi to users who are connected to Polygon? It seems like we might want to restrict this to mainnet.

File: src/pages/Home.vue [hunk 1]
```diff
@@ -55,20 +59,26 @@ import { useRouter } from 'vue-router';
...
+    PoolsTable,
+    FeaturedPools
   },
 
   setup() {
     // COMPOSABLES
     const router = useRouter();
-    const { isWalletReady, isV1Supported } = useWeb3();
+    const { isWalletReady, isV1Supported, appNetworkConfig } = useWeb3();
+    const isEthereum = computed(() =>
```
Comment (evgenyboxer) [inline src/pages/Home.vue:79]: We have something similar with EIP1559 support and @TomAFrench came up with a nice feature based solution.
Essentially you add "supportsElementPools" to each config and that's when the feature is active.

Check `supportsEIP1559` for reference.

File: src/components/sections/FeaturedPools.vue [hunk 2]
```diff
@@ -0,0 +1,75 @@
+<template>
+  <div>
+    <h3 class="mb-4">Featured ecosystem protocol</h3>
```
Comment (evgenyboxer) [inline src/components/sections/FeaturedPools.vue:3]: Shouldn't it be translated?

File: package-lock.json [hunk 3]
```diff
@@ -37270,6 +37270,7 @@
       "version": "4.5.12",
       "dev": true,
       "requires": {
+        "@babel/core": "^7.11.0",
```
[the remark this answers is not in the document]
Comment (1saf) [reply package-lock.json:37273]: sure

File: src/pages/Home.vue (hunk 1 above)
Comment (1saf) [reply src/pages/Home.vue:79]: ok thank

File: src/components/sections/FeaturedPools.vue (hunk 2 above)
Comment (1saf) [reply src/components/sections/FeaturedPools.vue:3]: Yeos

File: src/pages/Home.vue (hunk 1 above)
Comment (1saf) [reply src/pages/Home.vue]: resolved

File: src/components/sections/FeaturedPools.vue (hunk 2 above)
Comment (1saf) [reply src/components/sections/FeaturedPools.vue:3]: resolved

File: package-lock.json (hunk 3 above)
[the remark this answers is not in the document]
Comment (1saf) [reply package-lock.json:37273]: it didnt change anything

**Pull request merged**